app/services/ChargeCService.scala

```diff
@@ -35,17 +35,18 @@ object ChargeCService {
 
     def getEmployerDetails(index: Int): Option[(String, Boolean)] =
```

Comment:
This method can be made more concise by using flatMap with case mapping to Tuple, i.e. something like:-

ua.get(IsSponsoringEmployerIndividualPage(index)) flatMap {
        case true => ua.get(SponsoringIndividualDetailsPage(index)).map(i => Tuple2(i.fullName, i.isDeleted))
        case _ => ua.get(SponsoringOrganisationDetailsPage(index)).map(o => Tuple2(o.name, o.isDeleted))
      }